source/grid/grid_in.cc

```diff
@@ -2195,7 +2195,7 @@ GridIn<dim, spacedim>::read_msh(const std::string &fname)
 
   // Vertex renumbering, by dealii type
   const std::vector<std::vector<unsigned int>> gmsh_to_dealii = {
-    {{{0}},
+    {{0},
```

Comment:
?? Isn't this creating an array of length 0, instead of an array of length 1 with one zero entry?

I don't know that it would matter, but it looks strange. Separately, the outer `std::vector` might as well be a `std::array<...,8>`.

Reply:
~~~

int
main(void)
{
  initlog();
  const std::vector<std::vector<unsigned int>> gmsh_to_dealii = {
    {{0}, {{0, 1}}, {{0, 1, 2}}, {{0, 1, 3, 2}}}};

  for (const auto &i : gmsh_to_dealii)
    {
      deallog << "Size: " << i.size() << std::endl;
    }
}
~~~
produces
~~~
DEAL::Size: 1
DEAL::Size: 2
DEAL::Size: 3
DEAL::Size: 4
~~~
on my machine...

Reply:
Without this, I get the warning:
~~~
warning: braces around scalar initializer
~~~

Reply:
How bizarre... Thanks for double checking!